Hyper/Scene.cs

```diff
@@ -125,6 +108,31 @@ public bool TryFlipClosestCar(bool testOnly = false)
         return false;
     }
 
+    /// <summary>
+    /// Picks a lamp if it's close enough to the player.
+    /// </summary>
+    /// <param name="testOnly">If true the lamp won't be picked even if it could be</param>
+    /// <returns>True if a lamp was (or could be) picked. False otherwise.</returns>
+    public bool TryPickLamp(bool testOnly = false)
```

Comment:
What is the purpose of testOnly?

Reply:
Also I think this should be moved to a controller

Reply:
It's entirely analogous to other `TryDoSomething` methods in `Scene`: `testOnly` is used for displaying tiptext. We could move it from this class but then we probably should also move these other methods as well to keep it consistent. Should we do it?

Reply:
I don't like the fact that TryFlipCar is here but its up to you